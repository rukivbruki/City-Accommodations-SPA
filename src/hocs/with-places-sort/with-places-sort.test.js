import React from 'react';
import renderer from 'react-test-renderer';
import withPlacesSort from './with-places-sort';

const MockComponent = () => <div />;
const MockComponentWrapped = withPlacesSort(MockComponent);

it(`withPlacesSort snapshot`, () => {
  const tree = renderer.create(<MockComponentWrapped />).toJSON();
  expect(tree).toMatchSnapshot();
});
