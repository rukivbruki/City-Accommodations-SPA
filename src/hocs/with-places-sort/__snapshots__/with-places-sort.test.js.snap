// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withPlacesSort snapshot 1`] = `<div />`;
